DQMOffline/Trigger/python/HiggsMonitoring_cff.py

```diff
@@ -1,4 +1,141 @@
+
 import FWCore.ParameterSet.Config as cms
 
+from DQMOffline.Trigger.HiggsMonitoring_cfi import hltHIGmonitoring
+
+##############################DiLepton cross triggers######################################################
+mu23TrkIsoVVLEle8CaloIdLTrackIdLIsoVL_muref = hltHIGmonitoring.clone()
+mu23TrkIsoVVLEle8CaloIdLTrackIdLIsoVL_muref.nmuons = cms.uint32(1)
+mu23TrkIsoVVLEle8CaloIdLTrackIdLIsoVL_muref.nelectrons = cms.uint32(1)
+mu23TrkIsoVVLEle8CaloIdLTrackIdLIsoVL_muref.FolderName = cms.string('HLT/Higgs/DiLepton/HLT_Mu23_TrkIsoVVL_Ele8_CaloIdL_TrackIdL_IsoVL_muref/')
+mu23TrkIsoVVLEle8CaloIdLTrackIdLIsoVL_muref.numGenericTriggerEventPSet.hltPaths = cms.vstring("HLT_Mu23_TrkIsoVVL_Ele8_CaloIdL_TrackIdL_IsoVL_v*") 
```

Comment:
is this available in 2017 ?
I find HLT_Mu23_TrkIsoVVL_Ele12_CaloIdL_TrackIdL_IsoVL_DZ
could you please share the JIRA ticket, please ?